pkg/maps/policymap/trafficdirection.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2018 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package policymap
+
+// TrafficDirection specifies the directionality of policy (ingress or egress).
+type TrafficDirection uint8
+
+const (
+	// Egress represents egress traffic.
+	Egress TrafficDirection = 1
```

Comment:
This will sound super picky:
Can we change Egress to `0` and Ingress to `1`?

```
I/O -> Input / Output -> 1 / 0
Ingress / Egress -> Input / Output -> 1 / 0
```

Reply:
This will break the current datapath implementation cc: @joestringer .

Reply:
:-(

Reply:
For clarification, existing `PolicyMap` entries have a 0 bit in one of the padding fields which we're now going to use for this directionality. Since we only support ingress policies today, that bit is always 0 for current policies. Defining the Ingress direction as 0 means that these entries will be exactly the same as they are today.

If we delete the policymaps or flush their contents during upgrade then we could have more freedom on this.